feat: adding action yaml and readme (#1)
SilverbackLtd/build-action: README.md

> # silverbackltd-build

Reviewer: I believe this gets published to the Actions store page
```suggestion
 # Silverback Build Workflow
```

> SilverbackLTD Build workflow will automatically generate dockerfiles, build docker images and push those images to ghcr.

Reviewer: ```suggestion
Automatically generate dockerfiles (unless already commited), build docker images and push those images to a configured container registry like GHCR
```

## Outputs

### `build_images`
The images that are built and pushed by the workflow

## Example Usage
```
steps:

>   - name: silverback-build-push

Reviewer: ```suggestion
  - name: Build image
```

>     uses: SilverbackLtd/silverbackltd-build@v3

Reviewer: Use the current name, and we should release a v1 tag first
```suggestion
    uses: SilverbackLtd/build-action@v1
```

# Silverbackltd Build Workflow

Automatically generate dockerfiles (unless already committed), build docker images and push those images to a configured container registry like GHCR

## Outputs

### `build_images`
The images that are built and pushed by the workflow

> ## Example Usage

Reviewer: Nice, this looks perfect UX

> To push to a ghcr registry:

Reviewer: make sure to include permissions required

Author: ah, good call
